S32-io/IO-Socket-INET-UNIX.t

```diff
@@ -6,10 +6,12 @@ plan 8;
 if $*DISTRO.is-win {
     skip-rest 'UNIX socket support on Windows NYI';
 } else {
+    chdir $*TMPDIR;
+
     my IO::Socket::INET:_ $server;
     my IO::Socket::INET:_ $client;
     my IO::Socket::INET:_ $accepted;
-    my Str:D              $host      = $*TMPDIR.add("test-$*PID.sock").Str;
+    my Str:D              $host      = "./test-$*PID.sock";
```

Comment:
I'd suggest a bit less generic name here. However little the probability for an entry with the same name to exists in the temp dir, I'd rather make it even smaller. To my view, the 100% reliable solution (unless race condition is an option) is to add just one line:

```raku
$host = "./test-$*PID-" ~ ++$ ~ ".sock" while -e $host;
```

Reply:
Oh, there is a risk of race conditions when checking in [that way](https://en.wikipedia.org/wiki/Time-of-check_to_time-of-use), unfortunately.

Reply:
I figured existing I/O tests would already have dealt with this sort of problem. `S16-filehandles/io.t` opens files in the current directory and uses a less predictable filename:
```raku
sub nonce () { return ".{$*PID}." ~ (1..1000).pick() }
my $filename = 'tempfile_filehandles_io' ~ nonce();
```
But will silently continue with tests and remove the files if they already exist. I'm a bit wary of reusing this code though. Do we always have permissions to write there?